components/agenda/src/Agenda.svelte

```diff
@@ -1082,6 +1074,16 @@
                     <p>
                       {@html DOMPurify.sanitize(event.description ?? "")}
                     </p>
+                    {#if _this.display_metadata && event.metadata}
+                      {#each Object.entries(event.metadata) as entry}
+                        {@const key = entry[0]}
```

Comment:
I haven't seen this before! What does `@const` do here?

Reply:
It's some syntactic sugar; lets me reference `key` further within the `#each` block to hopefully make it clearer to others exactly what's going on.

The big benefit is for if/when we reference this multiple times, like `<li data-key={key}>{key}</li>` instead of `<li data-key={entry[0]}>{entry[0]}</li>`